### PR TITLE
No network type

### PR DESCRIPTION
**Pull request by mariano54**
Removes the concept of NetworkType, which was necessary before (when mainnet had transactions disabled). Now, there is no difference between different networks in terms of code, so this complexity can be removed permanently.

This has been tested manually with old configs, and it works fine.

### CODE REVIEW

File: chia/farmer/farmer_api.py [hunk 1]
```diff
@@ -30,6 +29,7 @@
 from chia.ssl.create_ssl import get_mozilla_ca_crt
 from chia.types.blockchain_format.pool_target import PoolTarget
 from chia.types.blockchain_format.proof_of_space import ProofOfSpace
+from chia.types.blockchain_format.sized_bytes import bytes32
```
Comment (xdustinface) [inline chia/farmer/farmer_api.py:32]:
```suggestion
```

**Review by xdustinface — DISMISSED** [review]
There are still a bunch of `NETWORK_TYPE` entries in `initial_config.yaml` and you missed to drop the `bytes32` import after your changes to use `selected_network`.

**Comment by jack60612** [comment]
is this ready for review?

**Review by jack60612 — DISMISSED** [review]
lgtm, glad you stripped it from the config

**Review by jack60612 — APPROVED** [review]
LGTM, that's probably a bit clearer

**Pull request merged**